tests/jobs/test_triggerer_job.py

```diff
@@ -448,7 +452,7 @@ def test_invalid_trigger(session, dag_maker):
     job.load_triggers()
 
     # Make sure it turned up in the failed queue
-    assert list(job.runner.failed_triggers) == [1]
+    assert len(job.runner.failed_triggers) == 1
```

Comment:
Should we still check `job.runner.failed_triggers[0][0]` is `1`?

Reply:
Perhaps – but all tests are heuristics I suppose. Wdyt?

Reply:
Fixed in c891e106905a864c3aa5da57cd8495449e4d3b65.